BUILD.md

```diff
@@ -547,25 +556,13 @@ See more information on the RKNPU Execution Provider [here](./docs/execution_pro
 
 ### DebugNodeInputsOutputs
 OnnxRuntime supports build options for enabling debugging of intermediate tensor shapes and data.
+
 #### Build Instructions
-##### Set onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=1
-Dump tensor input/output shapes for all nodes to stdout.
-```
-# Linux
-./build.sh --cmake_extra_defines onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=1
-# Windows
-.\build.bat --cmake_extra_defines onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=1
-```
-##### Set onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=2
-Dump tensor input/output shapes and output data for all nodes to stdout.
-```
-# Linux
-./build.sh --cmake_extra_defines onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=2
-# Windows
-.\build.bat --cmake_extra_defines onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=2
-```
-##### Set onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=0
-To disable this functionality after previously enabling, set onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=0 or delete CMakeCache.txt.
+|Set onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=1|Set onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=2|Set onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=0|
+|--|--|--|
+|Dump tensor input/output shapes for all nodes to stdout|Dump tensor input/output shapes and output data for all nodes to stdout|Disable this functionality after previously enabling|
+**Linux**<br> `./build.sh --cmake_extra_defines onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=1`<br><br>**Windows**<br>`.\build.bat --cmake_extra_defines onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=1`|**Linux**<br>`./build.sh --cmake_extra_defines onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=2`<br><br>**Windows**<br>`.\build.bat --cmake_extra_defines onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=2`| set `onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS=0` or delete CMakeCache.txt.|
+
```

Comment:
lot of repetitions, can we please make the command generic enough and then list  the three values onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS takes and their meaning? Also please consider listing in the order 0,1,2 instead of 1,2,0 and may be ONNXRUNTIME needs to be all in CAPs.

Reply:
Thanks. Addressed this and all other comments in #3976 (sorry, I messed up the commit history and created a new PR for clean it up)